include/wx/menuitem.h

```diff
@@ -163,6 +170,10 @@ class WXDLLIMPEXP_CORE wxMenuItemBase : public wxObject
     bool          m_isChecked;      // is checked?
     bool          m_isEnabled;      // is enabled?
 
+#if wxUSE_ACCEL
+    wxVector<wxString> m_extra_accels; // extra accels will work, but won't be shown in wxMenuItem title
```

Comment:
Let's use camel case for the members:
```suggestion
    wxVector<wxString> m_extraAccels; // extra accels will work, but won't be shown in wxMenuItem title
```

(also see the suggestion below for using a vector of accelerator entries directly).